oshi-core/src/main/java/oshi/jna/platform/mac/SystemB.java

```diff
@@ -388,6 +422,19 @@ public IFmsgHdr2(Pointer p) {
     int proc_pidpath(int pid, Pointer buffer, int buffersize);
 
     /**
+     * Return resource usage information for the given pid, which can be a live process or a zombie.
+     *
+     * @param pid
+     *            the process identifier
+     * @param flavor
+     *            the type of information requested
+     * @param buffer
+     *            holds results
+     * @returns 0 on success; or -1 on failure, with errno set to indicate the specific error.
+     */
+    int proc_pid_rusage(int pid, int flavor, Structure buffer);
```

Comment:
Should use type `RUsageInfoV2` here.  Or, if preferred, `RUsageInfoT` (define to extend `Structure`) which the V2 class can subsequently extend.